portugues/colabora/comdiferencas/comdiferencas.md

```diff
@@ -0,0 +1,2 @@
+Você encontra os dois commits que são o divisores de água entre o aparecimento do bug que tanto lhe incomoda. Você então:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.